src/Gateway/Events/ThreadListSync.php

```diff
@@ -17,7 +19,7 @@ class ThreadListSync
     public ?array $channel_ids;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\Channel[]
+     * @var Channel[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)